### ISSUE TITLE
Alternative Download

### ISSUE THREAD

**Issue by IzzySoft** [msasikanth/pinnit#25]
Would you consider making your great FOSS app available outside the walled garden of Google Play – e.g. by attaching the (signed) APK file to your tags/releases, or bringing the app to [F-Droid](https://f-droid.org/)? That way, poor Huawei users and others without Play Store access could use it as well :smiley:

Thanks in advance!

**Comment by IzzySoft** [comment]
Thanks for accepting my request! Is there any ETA? I see there was a new release tagged today, but again without an APK attached (with a signed APK available at the tag, I could take your app into [my repo](https://apt.izzysoft.de/fdroid) until you're ready for "F-Droid proper").

**Comment by IzzySoft** [comment]
Happy holidays! Just a little heads-up and looking if there might be something for my stockings? :santa: :gift: :smiley:

**Comment by msasikanth** [comment]
Yes, once scheduled notification feature is done I will work on adding APK with every GitHub releases. At this point, automating the releases to F-Droid seems like a bit of work, so for now, I will stick with GitHub releases.

**Comment by IzzySoft** [comment]
Thanks! As soon as APKs are available at tags here, I can add your app to my repo (as pointed out before). Automating releases to F-Droid wouldn't even require an APK to be available (just the tag, with the tag name corresponding to versionName – a condition that's already met: tag `v1.1.1` has versionName `1.1.1`, which meets the default pattern of `v%v` (i.e. `v<versionName>`).

Well, let's see. Once your APK is available and in my repo, we see what my library scanner says. If that indicates it should be good to go to F-Droid, I gladly help you with that if you want (disclosure: I'm one of the F-Droid maintainers).

**Comment by msasikanth** [comment]
I have released v1.2.0 of Pinnit and tags have been updated accordingly and added the release APK as well. Feel free to add my app to the F-Droid repo. Closing this issue for now.

**Comment by IzzySoft** [comment]
Thanks! Will [show up here](https://apt.izzysoft.de/fdroid/index/apk/dev.sasikanth.pinnit) in about 3.5h (around 7:45 CET/UTC+1) with the next sync. Looks good, and should be meeting F-Droid's strict inclusion criteria as far as I can see. So if you wish your app show up in that (more prestiguous) repo, be welcome to [file a Request for Packaging](https://gitlab.com/fdroid/rfp/issues) :smiley:

Until then, if you want to include a badge next to Play Store's, [I have this one](https://gitlab.com/IzzyOnDroid/repo/#if-my-app-is-available-here-do-you-have-a-badge-i-can-use-to-link-to-it) for you to link to your app in my repo :smile:

[<img src="https://gitlab.com/IzzyOnDroid/repo/-/raw/master/assets/IzzyOnDroid.png" width="250">](https://apt.izzysoft.de/fdroid/index/apk/dev.sasikanth.pinnit)

```markdown
[<img src="https://gitlab.com/IzzyOnDroid/repo/-/raw/master/assets/IzzyOnDroid.png" width="250">](https://apt.izzysoft.de/fdroid/index/apk/dev.sasikanth.pinnit)
```
